ReactNativeClient/lib/reducer.js

```diff
@@ -741,6 +792,15 @@ const reducer = (state = defaultState, action) => {
 			} else {
 				newState.notesParentType = 'Search';
 			}
+
+			// Update history when searching
```

Comment:
Missing test

Reply:
I'll add that. Also a test for Goto Anything.